docs/4.0/getting-started/download.md

```diff
@@ -10,7 +10,7 @@ toc: true
 
 Download ready-to-use compiled code for **Bootstrap v{{ site.current_version}}** to easily drop into your project, which includes:
 
-- Compiled and minified CSS bundles (default, grid-only, and Reboot-only)
+- Compiled and minified CSS bundles (see [comparative of css files]({{ site.baseurl }}/docs/{{ site.docs_version }}/getting-started/contents/#comparative-of-css-files))
```

Comment:
Wrong branch, please revert the unrelated changes and push them to a new branch which will only have those.

Reply:
Yes, I apologize. All the contributions I'm making to Bootstrap today is to learn how to use git. It will not happen again.

Reply:
No worries :)

The patches will be squashed into one when merge. /CC @mdo